### PR TITLE
add button constants; improve examples and doc

### PR DESCRIPTION
**Pull request by dhalbert**
- Add `ButtonPacket.B_1`, `B_2`, `B_3`, `B_4`, `UP`, `DOWN`, `LEFT`, `RIGHT`. 
- Remove crickit_test example, since it's now in a guide.
- Add AccelerometerPacket example.
- Improve documentation.

### CODE REVIEW

File: adafruit_bluefruit_connect/button_packet.py [hunk 1]
```diff
@@ -35,7 +35,25 @@
 from .packet import Packet
 
 class ButtonPacket(Packet):
-    """A packet containing a button name and its state"""
+    """A packet containing a button name and its state."""
+
+    B_1 = '1'
```
Comment (kattni) [inline adafruit_bluefruit_connect/button_packet.py]: If we need to call it something more than `1`, then I think we should call it `BUTTON_1`. It's clearer that way.

File: adafruit_bluefruit_connect/button_packet.py [hunk 2]
```diff
@@ -35,7 +35,25 @@
...
+    """Code for Button 1 on the Bluefruit LE Connect app Control Pad screen."""
+    B_2 = '2'
```
Comment (kattni) [inline adafruit_bluefruit_connect/button_packet.py]: Same.

File: adafruit_bluefruit_connect/button_packet.py [hunk 3]
```diff
@@ -35,7 +35,25 @@
...
+    """Button 2."""
+    B_3 = '3'
```
Comment (kattni) [inline adafruit_bluefruit_connect/button_packet.py]: Same.

File: adafruit_bluefruit_connect/button_packet.py [hunk 4]
```diff
@@ -35,7 +35,25 @@
...
+    """Button 3."""
+    B_4 = '4'
```
Comment (kattni) [inline adafruit_bluefruit_connect/button_packet.py]: Same.

**Comment by jerryneedell** [comment]
tested the PR on feather_nrf52840 ad particle_xenon -- examples worked well! Thanks!

File: adafruit_bluefruit_connect/button_packet.py (hunk 1 above)
Comment (dhalbert) [reply adafruit_bluefruit_connect/button_packet.py]: Should we call `UP`, etc. `BUTTON_UP`? Originally I did `ONE`, `TWO`, but that seemed kind of silly. `B1` was not accepted by pylint.

Comment (kattni) [reply adafruit_bluefruit_connect/button_packet.py]: Hmm. I feel like `UP` and `DOWN` etc. are clear. But perhaps for consistency?

**Pull request merged**